data/mm/world/overworld.yml

```diff
@@ -218,23 +224,23 @@
   locations:
     "Termina Field Water Chest": "has(MASK_ZORA)"
     "Termina Field Tall Grass Chest": "true"
-    "Termina Field Tree Stump Chest": "has(HOOKSHOT) || (has_beans && has_bottle)"
+    "Termina Field Tree Stump Chest": "has(HOOKSHOT) || can_use_beans"
     "Termina Field Kamaro Mask": "can_play(SONG_HEALING)"
     "Termina Field Tall Grass Grotto": "true"
     "Termina Field Pillar Grotto": "true"
-    "Termina Field Peahat Grotto": "can_fight || has(BOW)"
+    "Termina Field Peahat Grotto": "can_fight || has(BOW) || has(MASK_DEKU)"
     "Termina Field Dodongo Grotto": "has_weapon || has(BOMB_BAG) || has(MASK_GORON) || has(BOW)"
     "Termina Field Bio Baba Grotto": "can_break_boulders && has(MASK_ZORA)"
     "Termina Field Scrub": "event(SCRUB_TELESCOPE) && has(WALLET)"
     "Termina Field Gossip Stones HP": "has(OCARINA) && ((has(MASK_GORON) && has(SONG_GORON_HALF, 2)) || (has(MASK_DEKU) && has(SONG_AWAKENING)) || (has(MASK_ZORA) && has(SONG_ZORA))) && can_break_boulders"
 "Road to Southern Swamp":
   region: ROAD_TO_SWAMP
   exits:
-    "Swamp": "true"
+    "Swamp Front": "true"
     "Termina Field": "true"
     "Swamp Archery": "true"
   locations:
-    "Road to Southern Swamp HP": "true"
+    "Road to Southern Swamp HP": "has_weapon_range"
```

Comment:
It's possible to climb without killing the bat but I guess it should be a trick

Reply:
it is a trick in MM yes

Reply:
Allright then 👍